src/naarad/utils.py

```diff
@@ -625,7 +625,7 @@ def detect_timestamp_format(timestamp):
   :param string timestamp: the timestamp string for which we need to determine format
   :return: best guess timestamp format
   """
-  time_formats = {'epoch': re.compile(r'^[0-9]{10}$'), 'epoch_ms': re.compile(r'^[0-9]{13}$'), 'epoch_fraction': re.compile(r'^[0-9]{10}\.[0-9]{3}'),
+  time_formats = {'epoch': re.compile(r'^[0-9]{10}$'), 'epoch_ms': re.compile(r'^[0-9]{13}$'), 'epoch_fraction': re.compile(r'^[0-9]{10}\.[0-9]{3}$'),
```

Comment:
Yeah we should have it as 3 or more because you possibly could have 3 or 6 digits after the decimal right? (Milliseconds and nanoseconds and microseconds)

Can you have it with 3 or more?